trikScriptRunner/include/trikScriptRunner/trikScriptRunner.h

```diff
@@ -55,6 +55,10 @@ class TrikScriptRunner : public QObject
 	/// Adds custom initialization steps when creating script engine (useful when used from outside of the TrikRuntime).
 	void addCustomEngineInitStep(const std::function<void (QScriptEngine *)> &step);
 
+	/// Gets all methods from executive objects(brick, script, etc) from ScriptEngineWorker
```

Comment:
Actually it returns method names, not methods themselves. Missed whitespace near '('. Missed '.' after 'etc'.